### PR TITLE
Improve IncrementalPredictor 

### PR DESCRIPTION
**Pull request by JovanVeljanoski**
Small improvements in the `vaex.ml.sklearn.IncrementalPredictor`
- [x] An epoch is now on the level of the entire input DataFrame, not on a batch level.
- [x] Clean up of unused code.
- [x] Tests pass

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 # vaex-ml 0.7.1-dev
    * Performance
       * IncrementalPredictor uses parallel chunked support (2x speedup possible) [#515](https://github.com/vaexio/vaex/pull/515)
+      * IncrementalPredictor: epochs now iterate over the whole DataFrame instead on a batch level [#523](https://github.com/vaexio/vaex/pull/523)
```
Comment (maartenbreddels) [inline CHANGELOG.md]: not performance, fix

Comment (JovanVeljanoski) [reply CHANGELOG.md]: Ah yeah, indeed. I was just adding to the list.. Fixed now.

File: packages/vaex-ml/vaex/ml/sklearn.py [hunk 2]
```diff
@@ -199,41 +199,30 @@ def fit(self, df, target, progress=None, **kwargs):
...
-            X = np.array(chunks[:-1]).T  # the most efficient way acctually depends on the algorithm (row of column based access)
-            y = np.array(chunks[-1], copy=False)
-            # For reference: tqdm version was - would be nice to have an informative progress bar
-            # for j in tqdm(range(num_epochs), leave=False, desc='Iterating over batch...'):
-
-            # Loop over for each epoch
-            for j, epoch in enumerate(range(self.num_epochs)):
-                progressbar((i2 * self.num_epochs + j) / (N_total * self.num_epochs))
+
+        # Loop over for each epoch
```
Comment (maartenbreddels) [inline packages/vaex-ml/vaex/ml/sklearn.py]: These kinds of comments do not add anything, it says the same as the code below, while after refactoring it is often forgotten, and then lies. Often, when a comment reflects 1 line of code, it should be a more clear line of code without comment, or the comment should explain what is to come in the next N lines.

File: packages/vaex-ml/vaex/ml/sklearn.py [hunk 3]
```diff
@@ -199,41 +199,30 @@ def fit(self, df, target, progress=None, **kwargs):
...
+        for epoch in range(self.num_epochs):
+            # Loop iterating over the DataFrame
```
Comment (maartenbreddels) [inline packages/vaex-ml/vaex/ml/sklearn.py]: same

**Pull request merged**